server/stream.go

```diff
@@ -173,11 +185,46 @@ func (a *Account) AddStreamWithStore(config *StreamConfig, fsConfig *FileStoreCo
 	mset.pubAck = append(mset.pubAck, OK...)
 	mset.pubAck = append(mset.pubAck, fmt.Sprintf(" {\"stream\": %q, \"seq\": ", cfg.Name)...)
 
+	// Setup subscriptions
+	if err := mset.subscribeToStream(); err != nil {
```

Comment:
As soon as you subscribe, you may receive messages that would possibly not be de-duped, no? In other words, should (and is it possible) the mset.rebuildDedupe() function be called before this one?

Reply:
Geez, that is why I was moving it and moved it not far enough.. Will fix.

Reply:
ok moved and force pushed.